orderbook/src/api.rs

```diff
@@ -7,7 +7,7 @@ use warp::Filter;
 
 pub fn handle_all_routes(
     orderbook: Arc<OrderBook>,
-) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
+) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
```

Comment:
Small touch up because warp expects filters to return tuples and it's needed for the `map` in main to work.